doc/release-notes/release-8.sgml.in

```diff
@@ -68,7 +78,6 @@ This section gives an account of those changes in three categories:
 
 </descrip>
 
```

Comment:
The empty line removal (below this change request) is an out of scope formatting change that is not consistent with the previous `<sect>` formatting that still uses two empty lines before the opening element. Please undo.

```suggestion


```